meta-lhg-wpe/recipes-wpe/wpewebkit/wpewebkit_%.bbappend

```diff
@@ -0,0 +1,15 @@
+# removing commercial plugins
+RDEPS_EXTRA_remove = " \
+                      gstreamer1.0-plugins-ugly-mpg123 \
+                      gstreamer1.0-plugins-bad-hls \
+                     "
+
+# remove playready
+PACKAGECONFIG_remove = "playready"
+
+# add opencdm if optee detected
+PACKAGECONFIG_append = " ${@bb.utils.contains('MACHINE_FEATURES', 'optee', 'opencdm', '', d)}"
+
+PACKAGECONFIG[westeros] = "-DUSE_WPEWEBKIT_PLATFORM_WESTEROS=ON -DUSE_HOLE_PUNCH_GSTREAMER=OFF -DUSE_WESTEROS_SINK=OFF -DUSE_GSTREAMER_GL=ON,,westeros"
+
+CXXFLAGS_append_mx8 = " -DLINUX -DEGL_API_FB -DWL_EGL_PLATFORM"
```

Comment:
Ok for now, but it will be better if we can put these flags in a common *.inc (like some imx8-egl.inc) file and include it where ever required.